src/containers/SingleWorkout/SingleWorkout.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import timeIcon from './icons/clockicon.svg';
+import dumbbellIcon from './icons/trainingicon.svg';
+import { motion } from 'framer-motion';
+import { useHistory } from 'react-router-dom';
+
+const SingleWorkout = () => {
+  const history = useHistory();
+  const workout = {
```

Comment:
maybe you can store the workouts in a `.json` file and fetch them all and filter the one you want with the is you get, 
I know its not the best solution but for now it is, just to make the website dynamic

Reply:
I am not sure of this but I think we can not fetch local json files since it requires HTTP request. 
I will try to do it